src/components/Categories.jsx

```diff
@@ -0,0 +1,56 @@
+
+import { Component } from 'react';
+import Table from 'react-bootstrap/Table'
+import '../style/categories.css'
+class Categories extends Component {
+
+
+handleTransactions = (transactions) =>
+{
+    let categories = {}
+    transactions.map(t => {
+        categories[t.category]?categories[t.category] += t.amount: categories[t.category] = t.amount
+        return true;
+    })
+    return Object.entries(categories)
+}
+
+render()
+{
+  let transactions = this.props.transactions
+  transactions = this.handleTransactions(transactions)
+
+  return(
+    <div className="categories">
+<Table striped bordered hover>
```

Comment:
make sure you use correct indentation (alt+shift+F) and make your code more readable